### PR TITLE
remove all except github

### PR DESCRIPTION
**Pull request by lionelvillard**
## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,18 @@
-# Knative Eventing Contrib
+# Knative Eventing GitHub Source
```
Comment (mattmoor) [inline README.md:1]: Remove trailing whitespace:
```suggestion
# Knative Eventing GitHub Source
```

File: README.md [hunk 2]
```diff
@@ -1,32 +1,18 @@
...
-[![LICENSE](https://img.shields.io/github/license/knative/eventing-contrib.svg)](https://github.com/knative/eventing-contrib/blob/master/LICENSE)
-[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://knative.slack.com)
+|               |                                                                                                                    |
+| ------------- | ------------------------------------------------------------------------------------------------------------------ |
+| STATUS        | Alpha                                                                                                              |
+| Sponsoring WG | [Event Source](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#eventing-sources) |
 
-The Knative Eventing Contrib project provides source and channel implementations
-that:
+The Knative GitHub Source registers for events of the specified types on
```
Comment (mattmoor) [inline README.md:8]: Remove trailing whitespace:
```suggestion
The Knative GitHub Source registers for events of the specified types on
```

File: README.md [hunk 3]
```diff
@@ -1,32 +1,18 @@
...
-[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://knative.slack.com)
+|               |                                                                                                                    |
+| ------------- | ------------------------------------------------------------------------------------------------------------------ |
+| STATUS        | Alpha                                                                                                              |
+| Sponsoring WG | [Event Source](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#eventing-sources) |
 
-The Knative Eventing Contrib project provides source and channel implementations
-that:
+The Knative GitHub Source registers for events of the specified types on 
+the specified GitHub organization or repository, and brings those events into Knative.
```
Comment (mattmoor) [inline README.md:9]: Remove trailing whitespace:
```suggestion
the specified GitHub organization or repository, and brings those events into Knative.
```

File: README.md (hunk 1 above)
Comment (mattmoor) [inline README.md:1]: Format markdown:
```suggestion
# Knative Eventing GitHub Source
```

File: README.md [hunk 4]
```diff
@@ -1,32 +1,18 @@
...
+|               |                                                                                                                    |
+| ------------- | ------------------------------------------------------------------------------------------------------------------ |
+| STATUS        | Alpha                                                                                                              |
+| Sponsoring WG | [Event Source](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#eventing-sources) |
 
-The Knative Eventing Contrib project provides source and channel implementations
-that:
+The Knative GitHub Source registers for events of the specified types on 
+the specified GitHub organization or repository, and brings those events into Knative. 
+The GitHubSource fires a new event for selected GitHub event types.
```
Comment (mattmoor) [inline README.md:10]: Format markdown:
```suggestion
The Knative GitHub Source registers for events of the specified types on the
specified GitHub organization or repository, and brings those events into
Knative. The GitHubSource fires a new event for selected GitHub event types.
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

**Comment by lionelvillard** [comment]
@vaikas can you take a look?

File: OWNERS_ALIASES [hunk 5]
```diff
@@ -6,59 +6,6 @@ aliases:
...
-    - lberk
-
-  camel-approvers:
-    - nicolaferraro
-    - matzew
-  camel-reviewers:
-    - nicolaferraro
-    - matzew
-
-  github-approvers:
```
Comment (lberk) [inline OWNERS_ALIASES:24]: Is removing everybody but productivity & toc the owner alias that we really want?

Comment (lionelvillard) [reply OWNERS_ALIASES:24]: I think so since there is a top-level OWNERS file.

Comment (lberk) [reply OWNERS_ALIASES:24]: Ah sorry -- thought we were also flattening the structure as suggested in the camel pr

Comment (lionelvillard) [reply OWNERS_ALIASES:24]: it is flattened.

Comment (lberk) [reply OWNERS_ALIASES:24]: mea culpa!

**Comment by vaikas** [comment]
/lgtm
/approve

thanks!

**Pull request merged**